roles.py

```diff
@@ -6,12 +6,33 @@
 # In your Discord Server, make sure the role name is matching exactly and that
 # the bot role is above the role you want it to manage.
 
-VALORANT = 'Valorant'
-OVERWATCH = 'Overwatch'
-PROJECTWINTER = 'ProjectWinter'
+class Charon_Role:
+    def __init__(self, name=None, size=4, emoji='👍', imageURL=None):
+        self.name = name
+        self.size = size
+        self.emoji = emoji
+        self.imageURL = imageURL
+
+VALORANT = Charon_Role(
+    name='Valorant',
+    size=5, 
+    emoji='🔫',
+    imageURL='https://vignette.wikia.nocookie.net/leagueoflegends/images/8/88/Valorant_icon.png/revision/latest?cb=20200302105445')
+
+OVERWATCH = Charon_Role(
+    name='Overwatch',
+    size=6,
+    emoji='🎯',
+    imageURL='https://vignette.wikia.nocookie.net/overwatch/images/5/53/Pi_defaultblack.png/revision/latest/top-crop/width/220/height/220?cb=20160704195235')
+
+PROJECTWINTER = Charon_Role(
+    name='ProjectWinter',
+    size=8,
+    emoji='❄️',
+    imageURL='https://pbs.twimg.com/profile_images/1084835909185372160/QzmW7PC8.jpg')
 
 ROLES_LIST = [
-              VALORANT,
-              OVERWATCH,
-              PROJECTWINTER
-              ]
+    VALORANT,
```

Comment:
Are you using a linter? I've been using Flake8 if we want to standardize these indents and other stuff.

Reply:
I'm just using microsoft visual code, but i've been eyeballing it. i can probably find the Flake8 thing and use it.